src/ismn/interface.py

```diff
@@ -763,10 +763,15 @@ def get_dataset_ids(self, variable, min_depth=0, max_depth=0.1,
             else:
                 raise ValueError('Specified keyword \"{}\" not found in metadata! Use one of the following: {}'.format(k, lc_cl))
 
-        ids = np.where((self.metadata['variable'] == variable) &
-                       (self.metadata['depth_to'] <= max_depth) &
-                       (self.metadata['depth_from'] >= min_depth) &
-                       landcover_climate)[0]
+        if not max_depth:
```

Comment:
so now max_depth is optional, but min_depth is mandatory? Fine for me, but should be described in the doc string as well

Reply:
as you suggested, max_depth : float and None, optional (default: 0.1)